roles/edpm_network_config/templates/system_role_nmstate/single_nic_vlans.j2

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
Do we still create these templates keeping in mind the move to use inline templates that @bshephar is proposing in the other PR?

Reply:
Keeping templates as a separate file has some advantages. For example when it comes to questions of file history. If you want to know all the changes made to template, you just need to call git. But if the template is in line you get it mixed with everything else.

Reply:
@rabi i agree with you but at the moment we're still with files. When we'll remove other templates, I'll replace this one too. :)